cms/djangoapps/contentstore/views/item.py

```diff
@@ -1124,6 +1140,7 @@ def _compute_visibility_state(xblock, child_info, is_unit_with_changes, is_cours
     """
     Returns the current publish state for the specified xblock and its children
     """
+    child_info = None   # TODO remove when resolved
```

Comment:
This method was throwing exception, for now I have bypassed it, but I look at fixing later.